blocks_vertical/procedures.js

```diff
@@ -530,13 +530,32 @@ Blockly.ScratchBlocks.ProcedureUtils.updateDeclarationProcCode_ = function() {
   }
 };
 
+/**
+ * Focus on the last argument editor or label editor on the block.
+ * @private
+ */
+Blockly.ScratchBlocks.ProcedureUtils.focusLastInput_ = function() {
```

Comment:
...which leads to the question: should this be `focusLastEditor_`?

Up to you whether you want to change it.  Either way, LGTM.